stories/components/FileCollection/FileCollection.md

```diff
@@ -0,0 +1,109 @@
+
+Example:
+
+```javascript
+  {
+    name: 'file1.txt',
+    src: 'https://www.google.ca',
+    file_type: 'text/plain',
+    keep: true,
+  },
+```
+
+### Handling Errors
+
+Error is not handled in the component. This comes from the fact that we are tracking whether or not an uploaded file should be kept or deleted.
+Error can be handled by checking if `newFile[].length > 0 || !values.uploadedFiles.every(file => file.keep === false)`
+
+Example
+
+```javascript
+const fileCollection = () => {
+  const fileCollectionValid = values =>
+    values.files.length > 0 || !values.uploadedFiles.every(file => file.keep === false);
+
+  const handleSubmit= (formValues) => {
+    if (!fileCollectionValid(values)) {
+      return;
+    }
+    console.log(values);
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <FileCollection
+        newFiles={files} // initial empty []
+        data={uploadedFiles}
+        ...
+      />
+      {!fileCollectionValid(values) && (
+        <div style={{ color: 'red' }}>Please uploaded atleast one file</div>
+      )}
+    </form>
+  )
+}
+
+```
+
+## FileCollection
+
+Files can be dragged and dropped or browsed and muli-selected. Files array passed as prop pre-populate in a list.
+
+### Props
+
+| name            | type   | default | description                                                                                                                                         |
+| --------------- | ------ | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| data            | array  |         | Array of existing/default files as json object passed to pre populate.                                                                              |
+| newFiles        | array  |         | Pass value of an empty array you want to store uploading files in.                                                                                  |
+| onChange        | any    |         | An optional callback that is called for all inputs individually. Parameter returned is the an array of Files that are currently uploaded or dropped |
+| onDropVerbiage  | string |         | Verbiage for the dropzone when files are dragged on top to be dropped                                                                               |
+| dropFileVeriage | string |         | Verbiage for the dropzone when no files are uploaded                                                                                                |
+| browseVerbiage  | string |         | Verbiage for click to browse on dropzone                                                                                                            |
```

Comment:
nit: last | of each line isn't necessary and can also remove all spacing before it